conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/IllegalArgumentExceptionMapper.java

```diff
@@ -16,19 +16,23 @@
 
 package com.palantir.conjure.java.server.jersey;
 
-import com.codahale.metrics.Meter;
 import com.palantir.conjure.java.api.errors.ErrorType;
 import javax.ws.rs.ext.Provider;
 
 @Provider
 final class IllegalArgumentExceptionMapper extends JsonExceptionMapper<IllegalArgumentException> {
 
-    IllegalArgumentExceptionMapper(Meter internalErrorMeter) {
-        super(internalErrorMeter);
+    IllegalArgumentExceptionMapper(JerseyServerMetrics metrics) {
+        super(metrics);
     }
 
     @Override
     ErrorType getErrorType(IllegalArgumentException _exception) {
         return ErrorType.INVALID_ARGUMENT;
     }
+
+    @Override
+    InternalErrorCause getCause() {
+        return null;
```

Comment:
seems pretty sketchy to return null here since we assume non-null pretty widely in the code base. Perhaps an another variant to the enum?

Reply:
Fair point. I don't expect this to be called though. Could I throw an `UnsupportedOperationException` here instead ?

Reply:
I think i'd prefer you return something concrete so that a code change in the abstract class doesn't inadvertently break this implementation

Reply:
Ok, updated the class to be just `ErrorCause` and added an `OTHER` enum value, which we won't expect to be used.

Reply:
fly-by nit: perhaps `UNKNOWN` for uniformity with conjure?

Reply:
Changed to `UNKNOWN`